aemanalyser-core/src/main/java/com/adobe/aem/analyser/AemPackageConverter.java

```diff
@@ -143,7 +148,8 @@ public void convert(final Map<String, File> contentPackages) throws IOException,
 
         featuresManager.setAPIRegions(apiRegions);
         featuresManager.setExportToAPIRegion("global");
-
+        // populate with namespace mapping defaults, they can still be overridden from the bundle metadata
+        featuresManager.getNamespaceUriByPrefix().putAll(DEFAULT_NAMESPACE_MAPPINGS);
```

Comment:
You probably need a CLI parameter to set this when using cp2fm inside Cloud Manager. Otherwise this plugin uses different parameters than the actual conversion.

Reply:
Thanks for the note. I am not sure that this is the case. Without this change, https://github.com/adobe/aemanalyser-maven-plugin/tree/main/aemanalyser-maven-plugin/src/it/sling-initial-content-packages-test fails with the new CP Converter and now it passes.

Any Java API using `AemPackageConverter` is also going to benefit, but cannnot add overrides.

Which failure scenario do you see here?

Reply:
When inspecting the build image output for "recent" versions of AEM you will no longer find the output that you referenced in those slides. Implementation details can change :-)